keepassxc-browser/common/sites.js

```diff
@@ -69,6 +69,20 @@ const kpxcSites = {};
 kpxcSites.googlePasswordFormUrl = 'https://accounts.google.com/signin/v2/challenge/password';
 kpxcSites.savedForm = undefined;
 
+/**
+ * Tries to detect a username from the page.
+ * @returns {string}   Returns the detected username if there is one, undefined otherwise.
+ */
+kpxcSites.detectUsername = function() {
```

Comment:
Prefer this to be named `detectUsernameFromPage` (or similar) to be more explicit about where the information is coming from.

Reply:
Done.